google/cloud/spanner_dbapi/cursor.py

```diff
@@ -232,13 +243,18 @@ def description(self):
 
     @property
     def rowcount(self):
+        """Returns number of rows in the table.
+
+        :rtype: int
+        :returns: Number of rows.
+        """
```

Comment:
It returns somethings else: https://www.python.org/dev/peps/pep-0249/#rowcount

Reply:
This item is not resolved, there is still `number of rows in the table` on the 246 line